modules/govuk_elasticsearch/templates/es-backup-s3.erb

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+# Redirect stdout and stderr to syslog
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+S3_BUCKET=<%= @s3_bucket %>
+ES_REPO=<%= @es_repo %>
+ES_INDICES=<%= @es_indices %>
+AWS_REGION=<%= @aws_region %>
+AWS_ACCESS_KEY_ID=<%= @aws_access_key_id %>
+AWS_SECRET_ACCESS_KEY=<%= @aws_secret_access_key %>
+
+# indices: option allows us to specify indexes in a comma separated list rather than snap-shooting everything. However we can set
+# the $ES_REPO variable to '_all' to snapshot all indices.
+#
+# ignore_unavailable: option when true will cause indices that do not exist to be ignored during snapshot
+#
+# include_global_state: option will prevent the cluster global state from being stored as part of the snapshot.
+# The entire snapshot will fail if one or more indices participating in the snapshot don’t have all primary shards available
+
+
+/usr/bin/envdir <%= @env_dir %>/env.d /usr/bin/curl --connect-timeout 10 -sS -XPUT "http://127.0.0.1:9200/_snapshot/"${ES_REPO}"?wait_for_completion=true&pretty" -d '{
```

Comment:
Is this conflating the "create repository" call with the "create snapshot" call? I think this just creates the initial repository if it doesn't exist.

I expected the snapshot one to look like `/_snapshot/$ES_REPO/$SNAPSHOT_NAME`

https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html#_snapshot